projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-viewer.component.html

```diff
@@ -15,42 +15,59 @@
 </mv-comment-set-header>
 <mv-participants-list></mv-participants-list>
 
-<div mvGrabNDrag [dragX]="pdfViewer" [dragEnabled]="enableGrabNDrag"
-     #viewerContainer
+<div
      class="pdfContainer"
      [ngStyle]="{ height: height }"
-     [ngClass]="{ 'pdfContainer': true, 'annotations': enableAnnotations, hidden: errorMessage, 'show-comments-panel': (showCommentsPanel || showIcpParticipantsList), 'grabNDrag': enableGrabNDrag }">
-  <div #pdfViewer
-       class="pdfViewer"
-       mvCreateTextHighlight
-       [ngClass]="{ hidden: loadingDocument, highlightMode: highlightMode | async, drawMode: drawMode | async }">
+     [ngClass]="{ 'pdfContainer': true, hidden: errorMessage }">
+
+  <div
+    #viewerContainer
+    mvGrabNDrag
+    [dragX]="viewerContainer"
+    [dragEnabled]="enableGrabNDrag"
+    id="viewerContainer"
+    class="viewer-container"
+    (scroll)="scrollTwo.scrollTop = viewerContainer.scrollTop"
+    [class.annotations]="enableAnnotations"
+    [class.show-comments-panel]="(showCommentsPanel || showIcpParticipantsList) && !(toolbarEvents.redactionMode | async)"
+    [class.grabNDrag]="enableGrabNDrag">
+
+    <div #pdfViewer
+         class="pdfViewer"
+         mvCreateTextHighlight
+         [ngClass]="{ hidden: loadingDocument, highlightMode: highlightMode | async, drawMode: drawMode | async }">
+    </div>
+
+    <mv-redactions *ngIf="(toolbarEvents.redactionMode | async); else annotationTemplate"
+                   [zoom]="zoom"
+                   [rotate]="rotation"
+    ></mv-redactions>
+
+    <ng-template #annotationTemplate>
+      <mv-metadata-layer
+        *ngIf="enableAnnotations && annotationSet"
+        [zoom]="zoom"
+        [rotate]="rotation">
+      </mv-metadata-layer>
+
+      <mv-bookmark-icons [zoom]="zoom"
+                         [rotate]="rotation">
+      </mv-bookmark-icons>
+    </ng-template>
   </div>
 
-  <mv-redactions *ngIf="(toolbarEvents.redactionMode | async);else annotationTemplate"
-                 [zoom]="zoom"
-                 [rotate]="rotation"
-  ></mv-redactions>
-
-  <ng-template #annotationTemplate>
-    <mv-metadata-layer
-      *ngIf="enableAnnotations && annotationSet"
-      [zoom]="zoom"
-      [rotate]="rotation">
-    </mv-metadata-layer>
+  <div #scrollTwo class="comments">
     <mv-comment-set #commentsPanel
```

Comment:
would something like this not work? [scrollTop]="viewerContainer.scrollTop"

Reply:
Potentially, yes but we want to scroll commentsPanel when it's visible and the viewer is scrolled. not on every digest cycle